spec/requests/posts_controller_spec.rb

```diff
@@ -1815,6 +1813,58 @@
       expect(public_post.custom_fields[Post::NOTICE_TYPE]).to eq(nil)
       expect(public_post.custom_fields[Post::NOTICE_ARGS]).to eq(nil)
     end
+
+    describe 'group moderators' do
+      fab!(:group) { Fabricate(:group) }
+
+      before do
+        SiteSetting.enable_category_group_moderation = true
+        topic.category.update!(reviewable_by_group_id: group.id)
+
+        sign_in(user)
+      end
+
+      it 'can create and remove notices as a group moderator' do
+        GroupUser.create!(group_id: group.id, user_id: user.id)
+
+        expect(Guardian.new(user).can_review_topic?(topic)).to eq(true)
```

Comment:
Just curious, why do we need this assertion?

Reply:
I think that was remnants of some dev work. You're right - it should be there. Removing.